### PR TITLE
added new endpoint

### PR DESCRIPTION
**Pull request by sandman45**
and new stream path

new stream path does NOT create the video Obj

### CODE REVIEW

File: api/controllers/radiant-info.js [hunk 1]
```diff
@@ -19,4 +21,25 @@ function getInfo(req, res, next) {
...
+
+function getVideoUrl(req, res, next) {
+    const buckets = {
+        DEV: process.env.DEV_S3_BUCKET,
+        STAGING: process.env.STAGING_S3_BUCKET,
+        PRODUCTION: process.env.PRODUCTION_S3_BUCKET,
+    };
+
+
+    const videoUrl = `https://s3.${process.env.S3_REGION}.amazonaws.com/${buckets[process.env.ENV]}/${req.params.uuid}-i.m3u8?params=true`;
```
Comment (laynemoseley) [inline api/controllers/radiant-info.js]: you know what might be really useful in this api? If you returned the status of those urls.

For example:

```
{
    videoUrl: { url: someurl, status: Uploaded },
    thumbnailUrl: { url: someurl, status: Pending }
}
```

You don't have to keep the state, you can just do a `HEAD` on the object in s3 to see if they are there already or not.

Comment (sandman45) [reply api/controllers/radiant-info.js]: hmm good idea..

Comment (sandman45) [reply api/controllers/radiant-info.js]: what happens if it fails uploading the thumb or video.. and its Pending Forever?

Comment (laynemoseley) [reply api/controllers/radiant-info.js]: hrmmm maybe you do need a database :D

**Pull request merged**